client/gutenberg/extensions/map/editor.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+
+import { settings } from './settings.js';
+import edit from './edit';
+import save from './save';
+import './style.scss';
+import './editor.scss';
+
+registerBlockType( settings.name, {
+	title: settings.title,
+	icon: settings.icon,
+	category: settings.category,
+	keywords: settings.keywords,
+	attributes: settings.attributes,
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+		if ( -1 !== settings.validAlignments.indexOf( align ) ) {
```

Comment:
Today I learned about `getEditWrapperProps` 🎉  Seems to be undocumented.

We could simplify this a bit, how about:
```js
getEditWrapperProps( { align } ) {
  if ( settings.validAlignments.includes( align ) ) {
    return { 'data-align': align };
  }
}
```

Reply:
I think `.includes()` is just newer JS that will need to be polyfilled by Babel and using the older method will just save some kilobytes because of no need to ship the polyfill.

Ok to leave as is :-)